### PR TITLE
1.2

### PR DESCRIPTION
**Pull request by CrimsonVoid**

### CODE REVIEW

File: hanyuu_commands.py [hunk 1]
```diff
@@ -432,14 +439,18 @@ def formatDate(dt):
         query = int(query);
         try:
             song = manager.Song(id=query)
-            message = [u"{c4}{meta} {c3}({trackid}){c} (LP: {c5}{lp}{c}) (R: {c5}{lr}{c})"\
-               .format(meta=song.metadata, trackid=song.id, lp=formatDate(song.lpd), lr=formatDate(song.lrd), **irc_colours)]
+            message = [u"{col_code}{meta} {c3}({trackid}){c} (LP:{c5}{lp}{c})"\
+                .format(col_code=irc_colours['c3' if song.requestable() else 'c4'],\
```
Comment (Wessie) [inline hanyuu_commands.py]: Song.requestable is a property, not a method

File: hanyuu_commands.py [hunk 2]
```diff
@@ -432,14 +439,18 @@ def formatDate(dt):
...
+                    meta=song.metadata, trackid=song.id, \
+                    lp=format_date(song.lpd), **irc_colours)]
         except (ValueError):
             message = []
     except (ValueError):
-        message = [u"{c4}{meta} {c3}({trackid}){c} (LP: {c5}{lp}{c}) (R: {c5}{lr}{c}) "\
-            .format(meta=song.metadata, trackid=song.id, lp=formatDate(song.lpd), lr=formatDate(song.lrd), **irc_colours) for \
-            song in manager.Song.search(query)]
+        message = [u"{col_code}{meta} {c3}({trackid}){c} (LP:{c5}{lp}{c})"\
+            .format(col_code=irc_colours['c3' if song.requestable() else 'c4'],\
```
Comment (Wessie) [inline hanyuu_commands.py]: Song.requestable is a property, not a method

File: hanyuu_commands.py [hunk 3]
```diff
@@ -414,12 +414,19 @@ def lucky(server, nick, channel, text, hostmask):
...
             return 'Never'
         else:
             dt = datetime.now() - dt
-            return '{d}d{h}h'.format(d=dt.days, h=dt.seconds/3600)
+            
+            if dt.total_seconds < 86400:
+                return '{h}h{m}m'.format(h=dt.total_seconds/3600, \
+                    m=(dt.total_seconds//3600)/60)
+            elif dt.days > 30
+                return '{m}m{d}d'.format(m=dt.days/30, d=dt.days//30)
```
Comment (Wessie) [inline hanyuu_commands.py]: Why are you doing dt.days/30 for both the amount on months and days

Comment (CrimsonVoid) [inline hanyuu_commands.py]: Because I'm a baka

**Pull request merged**